styleguide/_themes/derek/scss/patterns/grid/businessResource.scss

```diff
@@ -29,23 +40,15 @@
       }
     }
   }
+}
 
-  a {
-    color: $gray-darker;
-    font-weight: 600;
-    margin-top: 1em;
-    padding-bottom: 20px;
-    text-decoration: none;
-  }
-
-  img {
-    display: inline-block;
-    margin-right: 10px;
-    padding-top: 10px;
-    width: 60px;
+.businessResource-third {
+  @include make-xs-column(12);
+  @include make-md-column(4);
+  @include flex-box;
```

Comment:
We did that with ias, it doesn't do anything here but I wasn't sure why you suggested it there.